src/test/utils_check.cpp

```diff
@@ -277,5 +284,87 @@ TEST(utils_check, try_cached_as_check_what_message_test) {
 		if( msg.find(test_sentence) == string::npos )
 			FAIL();
 	}
+}
+
+TEST(utils_check, reasonable_size_string_test) {
+	{
```

Comment:
Test object are big, so all objects are created in separated scopes to free memory after test when object is not used anymore